server/client-config/routes.js

```diff
@@ -1,9 +1,31 @@
+const logger = require('@pubsweet/logger')
 const config = require('config')
 const { pickBy } = require('lodash')
+const DBExists = require('@pubsweet/db-manager/src/helpers/db-exists.js')
```

Comment:
Note that `dbExists` is exported as a function from the package root so you can avoid a fragile deep require.